package.json

```
@@ -26,6 +26,21 @@
     "test": "yarn workspaces run test",
     "test:ci": "yarn workspaces run test:ci"
   },
+  "simple-git-hooks": {
+    "pre-commit": "yarn lint-staged"
+  },
+  "lint-staged": {
+    "*.{js,ts}": [
+      "eslint --fix",
+      "prettier --write"
```

Our ESLint config includes a `prettier` plugin:
```suggestion
      "eslint --fix"
```